schemas/threat.yml

```diff
@@ -29,8 +29,8 @@
       type: keyword
       short: Threat tactic.
       description: >
-          Name of the type of tactic used by this threat. You can use the Mitre ATT&CK Matrix Tactic categorization, for example.
-          (ex. https://attack.mitre.org/tactics/TA0040/ )
+          Name of the type of tactic used by this threat. You can use the MITRE ATT&CK® Matrix tactic categorization, for example.
```

Comment:
I would switch the phrasing "use a" instead of "use the" for this sentence as well.

Reply:
thank you! this is why we peer review 😆